client/src/reducers/createdGroup.js

```diff
@@ -0,0 +1,12 @@
+import { CREATE_GROUP_SUCCESS ,RESET_CREATE_GROUP_STATUS } from '../actions';
+
+export default (state = null, action) => {
+  switch (action.type) {
+    case CREATE_GROUP_SUCCESS:
```

Comment:
Don't you think state will be mutated here? Make use of es6 spread operator to add new state.

Reply:
Do the  same for all reducers